lib/cursor.js

```diff
@@ -236,7 +236,9 @@ const hasNext = (self, callback) => {
   if (self.s.currentDoc) {
     return callback(null, true);
   }
-
```

Comment:
nit: I prefer the space before and after this check to make it very easy to scan when reading through code